pkgs/top-level/release-lib.nix

```
@@ -163,19 +163,26 @@ let
     (addMetaAttrs { maintainers = crossMaintainers; });
 
 
-  /* Recursively map a (nested) set of derivations to an isomorphic
-     set of meta.platforms values. */
-  packagePlatforms = mapAttrs (name: value:
+  /* Recursive for packages and apply a function to them */
+  recursiveMapPackages = f: mapAttrs (name: value:
       if isDerivation value then
-        value.meta.hydraPlatforms
-          or (subtractLists (value.meta.badPlatforms or [])
-               (value.meta.platforms or supportedSystems))
+        f value
       else if value.recurseForDerivations or false || value.recurseForRelease or false then
-        packagePlatforms value
+        recursiveMapPackages f value
```

Apologies if I'm missing something, but does the "f" do anything?
```suggestion
        recursiveMapPackages value
```

---

It's a bit confusing, but `recursiveMapPackages` takes two arguments, because it's defined like `recursiveMapPackages f = mapAttrs (...)`, which is the same as `recursiveMapPackages f attrs = mapAttrs (...) attrs`. So here it's called recursively with the same `f` but a different `attrs`.